### ISSUE TITLE
provide anvil-collect-debug-data tool

### ISSUE THREAD

**Issue by fabbione** [ClusterLabs/anvil#342]
```
18:03 < fabbione> https://github.com/kronosnet/ci-tools/blob/main/ci-setup-anvil-bm-vm#L219
18:03 < fabbione> and down
18:03 < fabbione> this is what I am collecting from CI
18:03 < fabbione> etc/hosts
18:03 < fabbione> anvil.log
18:03 < digimer> leave that alone, I'll make a tool
18:03 < fabbione> cloud-init*
18:03 < digimer> I like that idea a lot
18:04 < fabbione> and from the strikers I get the db dump from outside
18:04 < fabbione> i won´t touch it, but i´d like your tool to collect those bits as well
18:04 < digimer> yup, will do
18:04 < digimer> I'll make the tool from the perspective of collecting real-client/user data
18:04 < fabbione> exactly
18:04 < digimer> so I'll assume it has to collect everything useful
18:06 < fabbione> yeps
18:06 < fabbione> but please include the cloud-init* stuff
18:06 < fabbione> that´s only for CI
18:06 < fabbione> but it saves me the trouble to keep it separated in CI
18:06 < digimer> I will try to remember to do that
18:07 < fabbione> go over the list of files there ;)
```

**Comment by fabbione** [comment]
Blocked by https://github.com/ClusterLabs/anvil/issues/367

**Comment by digimer** [comment]
Ah, I forgot to collect cloud-init-*, I'll add that.

**Comment by fabbione** [comment]
striker-collect-debug --help reports that the tool is called striker-collect-data

**Comment by fabbione** [comment]
Once you add collection of cloud-* and fix the --help data, this bug can be closed.

CI is now using this tool to collect debug info.

You might notice that CI executes this command from both striker01 and 02. This is done on purpose in the event the DB is out of sync or one striker is down.

**Comment by digimer** [comment]
Should be resolved in pr#376

**Comment by fabbione** [comment]
verified in pr#376
